core/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -229,7 +230,7 @@ public XContentBuilder toXContentEmbedded(XContentBuilder builder, Params params
         builder.field(Fields.FOUND, exists);
 
         if (source != null) {
-            XContentHelper.writeRawField("_source", source, builder, params);
+            XContentHelper.writeRawField(Fields._SOURCE, source, builder, params);
```

Comment:
These `Fields` objects have fallen out of favor. In this case I'd go with something like `SourceFieldMapper.NAME` instead.